### PR TITLE
Revert hack

### LINKED ISSUE #644: Remove structlog initialization hack when new version is released
Once hynek/structlog#72 is released, we can revert #633.

### PR DESCRIPTION
**Pull request by glasserc**
Fixes #644 by reverting #633.

r? @leplatrem @Natim

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@ simplejson==3.8.2
 six==1.10.0
 SQLAlchemy==1.0.13
 statsd==3.2.1
-structlog==16.0.0
+structlog==16.1.0
```
Comment (Natim) [inline requirements.txt]: Maybe we need to add ``structlog>=16.1`` in setup.py too

Comment (glasserc) [inline requirements.txt]: OK, done in 796d6133225865e4150932509a8c483a696fedc0

**Comment by Natim** [comment]
Seems good to me.

**Comment by leplatrem** [comment]
r+

**Pull request merged**